src/main/java/blusunrize/immersiveengineering/common/util/compat/jei/bottlingmachine/BottlingMachineRecipeCategory.java

```diff
@@ -47,9 +47,11 @@ public void setRecipe(IRecipeLayoutBuilder builder, BottlingMachineRecipe recipe
 		builder.addSlot(RecipeIngredientRole.INPUT, 1, 13)
 				.addItemStacks(Arrays.asList(recipe.input.getItems()))
 				.setBackground(JEIHelper.slotDrawable, -1, -1);
-		builder.addSlot(RecipeIngredientRole.OUTPUT, 101, 13)
-				.addItemStack(recipe.output.get())
-				.setBackground(JEIHelper.slotDrawable, -1, -1);
+		int i = 0;
+		for(ItemStack out : recipe.output.get())
+			builder.addSlot(RecipeIngredientRole.OUTPUT, 101, 13+(i++)*18)
```

Comment:
IMO a traditional indexed for loop would be more readable here, using the return value of pre/post-increment is always a bit confusing.

Reply:
Fair